src/context/ProductsFilterContext.jsx

```diff
@@ -0,0 +1,13 @@
+const ProductFilterContext = createContext()
+const filter_product_reducer = (state, { type, payload }) => {
+    switch (type) {
+        case "SEARCH":
+            return { ...state, }
+        case "FILTERBY":
+            return;
+        case "SortBY":
```

Comment:
suggestion: to keep consistency, do all caps on SortBY as well

Reply:
corrected